_component_design/buttons-bar.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Button Bar
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A button bar consolidates multiple simple buttons into one multi-button component. You can combine multiple simple buttons and a single drop-down button.
+preview-image: preview-images/buttons-bar.svg
+status: stable
+resource: true
+last-modified: 2018-09-20
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a button bar in the following situations:
+
+- When there is a primary action represented by a simple button and you want to list additional actions in a drop-down menu.
+- To offer a number of action buttons at once and conserve horizontal space.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+In addition to the [Simple Buttons]({{site.baseurl}}/components/simple-buttons.html) best practices,
+consider the following best practices when you construct a button bar:
+
+- Do not use a button bar for objects with a small number of actions. For example, do not use a button bar for table objects that have only a create and delete action. When a table object has a small number of actions, use table controls at the top of the table. For more information, see [tables]({{site.baseurl}}/components/tables.html).
+- Do not mix button weights or sizes in a button bar.
+- Place the drop-down button as the last button in the bar.
+- Destructive actions should be placed inside the drop-down menu, and not as a visible button in the bar.
```

Comment:
Do not include destructive actions in a button bar unless they are in the dropdown menu of actions. This is because most of the time the destructive action will be disabled.